docs/modules/record/README.md

```diff
@@ -1,32 +1,85 @@
-# iriscli record
+# Record User Guide
 
-## Description
+## Basic Function Description
 
-Record allows you to submit, query and download your records on the chain.
+1. On-Chain record for the text data
+2. On-Chain record for the text file (TODO)
+3. On-Chain governance for the record params (TODO)
 
-## Usage
+## Interaction Process
 
-```shell
-iriscli record [command]
+### Record process
+
+1. Any users can initiate a record request. It will cost you some tokens. If there’s no record of the data on the existing chains, the request will be completed successfully and the relevant metadata will be recorded onchain. And you will be returned a record ID to confirm your ownership of the data.
+2. If any others initiate a record request for the same data, the request will be directly rejected and it will hint that the relevant record data has already existed.
+3. Any users can search/download onchain based on the record ID.
+4. At present, the maximum amount of stored data at most 1K Bytes. In the future, the dynamic adjustment of parameters will be implemented in conjunction with the governance module.
```

Comment:
Add description about the architecture of the record module. Please include these points:
1. Where the recording payload will be saved?
2. What is the metadata and how to generate it?
3. Why do we adopt this strategy?

Reply:
My suggestions are as follows:
1. Record payload is saved on the target chain as the statement "**the metadata of record payload will be recorded on the target chain**" says. So I think maybe there's no need to emphasize this again.
2. I will introduce the meanings of each component in the record metadata.
3. Record module which enables generating corresponding metadata from uploaded data on the target chain, and it's public and can NOT be tampered with. And that's the way you declare data sovereignty by blockchain.

Reply:
Agreed!